<ui version="4.0" >
 <class>LabelCreator</class>
 <widget class="QDialog" name="LabelCreator" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>562</width>
    <height>162</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <widget class="Kst::LabelBuilder" native="1" name="_labelText" />
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout" >
     <item row="0" column="0" >
      <widget class="QLabel" name="_Label_11" >
       <property name="text" >
        <string>Font &amp;size offset:</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buddy" >
        <cstring>_labelFontScale</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2" >
      <widget class="QDoubleSpinBox" name="_labelFontScale" >
       <property name="decimals" >
        <number>1</number>
       </property>
       <property name="minimum" >
        <double>-25.000000000000000</double>
       </property>
       <property name="singleStep" >
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="_Label_12" >
       <property name="text" >
        <string>Font famil&amp;y:</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buddy" >
        <cstring>_family</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="5" >
      <widget class="QFontComboBox" name="_family" />
     </item>
     <item row="0" column="5" >
      <widget class="Kst::ColorButton" name="_labelColor" >
       <property name="toolTip" >
        <string>Color</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3" >
      <widget class="QPushButton" name="_bold" >
       <property name="toolTip" >
        <string>Bold</string>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="shortcut" >
        <string>Alt+B</string>
       </property>
       <property name="checkable" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="4" >
      <widget class="QPushButton" name="_italic" >
       <property name="toolTip" >
        <string>Italic</string>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="shortcut" >
        <string>Alt+I</string>
       </property>
       <property name="checkable" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Kst::ColorButton</class>
   <extends>QToolButton</extends>
   <header>colorbutton.h</header>
  </customwidget>
  <customwidget>
   <class>Kst::LabelBuilder</class>
   <extends>QWidget</extends>
   <header>labelbuilder.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>LabelCreator</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>LabelCreator</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
